src/tutorials/0008-introduction-to-libp2p/05.md

```diff
@@ -0,0 +1,80 @@
+---
+    title: "Multiple environments supported"
+    type: "text"
```

Comment:
We can also add a quiz here regarding a libp2p module. Thoughts?

Reply:
Do you have anything in mind?

Reply:
What libp2p modules are required? if we add that to the test. Otherwise we can add something more specific to a transport as we go through that example

Reply:
done! 45a4b46a86fb59fb1cd3b5ea112386b96ae11b38

Added a very easy one about the initial part of the tutorial since I feel like talking about the libp2p modules would be a bit hard for some folks.
Let me know what you think!

Reply:
looks good!